index.html

```diff
@@ -369,24 +370,80 @@ <h1>Social Media Monitoring</h1>
+	</div>
+	<div class="row">
+    	<div class="col-md-9"></div>
+        <div class="col-md-3"><img src="img/OSINTlog.png" alt="Socmint" class="img-thumbnail"><h5>An OSINT log including social media posts, obtained by the ACLU via <a href="https://www.documentcloud.org/documents/6567425-Aclu-1-Reviewed-Redacted.html">FOIA.</a></h5></div>
     </div>
   	<div class="row">
     <h3> <span class="glyphicon glyphicon-eye-open"></span> Capabilities </h3>
-    <p> Users of these tools are able to look at geotagged posts over a range of social media services including Twitter and Instagram. In addition to <a href="https://www.inverse.com/article/8358-your-social-media-posts-are-fueling-the-future-of-police-surveillance">geographic based monitoring</a>, keyword based monitoring is also performed. </p>
-    <h3> <span class="glyphicon glyphicon-wrench"></span> Efficacy </h3>
-    <p>Unknown. </p>
+    <p>CPD Open Source Intelligence (OSINT) officers are responsible for compiling publicly accessible information (e.g. social media posts) related to investigations. Logs produced by OSINT officers on individual subjects often include links to social media accounts and even posts.</p>
+	<p>Geofeedia can associate social media posts with locations. After being called out by the ACLU, however, many social media platforms have <a href="https://www.aclunc.org/blog/facebook-instagram-and-twitter-provided-data-access-surveillance-product-marketed-target">restricted</a> Geofeedia’s access to their users’ data.</p>
+	<p>Dunami has been used by <a href="https://www.revealnews.org/blog/facebook-reactions-are-a-gift-to-advertisers-and-law-enforcement-alike/">FBI and CBP</a> “to determine networks of association, centers of influence and potential signs of radicalization.” Dunami has been used by CPD and CPS to monitor for “signs of violence and gang membership” among <a href="https://www.propublica.org/article/chicago-public-schools-social-media-monitoring-violence-gangs">students</a>.</p>
+	<h3> <span class="glyphicon glyphicon-wrench"></span> Efficacy </h3>
+    <p>CPD’s <a href="http://directives.chicagopolice.org/directives/">social media policy</a> claims that social media is a valuable tool for investigating crimes and gang activity, although the efficacy of social media monitoring is unclear.</p>
+	<p>The University of Chicago Crime Lab, which helped implement CPS’ social media monitoring program, <a href="https://www.wbez.org/stories/chicago-schools-monitoring-student-social-media-see-violence-drop/d0fbe47d-7a18-44b9-a276-38ca930833db">found</a> that schools which used social media monitoring experienced greater drops in misconduct and suspensions than those that didn't, although it did not find a significant difference in incidents of gun violence between these two sets of schools.</p>
```

Comment:
I am not sure if found is the proper term here. individual feedback for that on slack.